source/_integrations/compit.markdown

```diff
@@ -0,0 +1,68 @@
+---
+title: Compit
+description: Instructions on how to integrate Compit devices within Home Assistant.
+ha_category:
+  - Climate
+  - Number
+  - Select
+  - Sensor
+  - Switch
+  - Water heater
+ha_release: 2024.12
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - "@CompitHomeAssistant"
+  - "@Przemko92"
+ha_domain: compit
+ha_platforms:
+  - climate
+  - number
+  - select
+  - sensor
+  - switch
+ha_integration_type: integration
+---
+
+This integration is created by [Compit](https://compit.pl/) to integrate air conditioning, ventilation and heating controllers with Home Assistant. This integration needs Compit iNext account setup at [inext.compit.pl](https://inext.compit.pl).
+
+## Prerequisites
+
+- Create an account on [inext.compit.pl](https://inext.compit.pl).
+- Configure your Compit devices within the account.
+
+## Supported devices
+
+**This component supports the following Compit devices.**
+
+| Device        | Type                       | Description                                                                                                                                                                   |
+| ------------- | -------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Nano Color 2  | Room thermostat            | [Product page](https://compit.pl/produkty/termostaty-pokojowe/88-termostat-pokojowy-nano-color-2.html?ic=1)                                                                   |
```

Comment:
```suggestion
- [Nano Color 2](https://compit.pl/produkty/termostaty-pokojowe/88-termostat-pokojowy-nano-color-2.html?ic=1) - Room thermostat
```
We can use a list instead to avoid using a table here. The link can be directly on the device name